modules/apps/27-interchain-accounts/keeper/account.go

```diff
@@ -22,7 +22,7 @@ func (k Keeper) InitInterchainAccount(ctx sdk.Context, connectionID, counterpart
 		return err
 	}
 
-	if k.IsBound(ctx, portID) {
+	if k.portKeeper.IsBound(ctx, portID) {
```

Comment:
Based off of #288, is this the intended usage? @seantking @colin-axner 

I've added `IsBound` to expected keeper interface and re-able the previously commented out test.

Reply:
Yes it is :+1: Thanks!